net-libs/libsrtp/libsrtp-1.6.0.ebuild

```diff
@@ -27,7 +27,7 @@ DOCS=( CHANGES README TODO )
 MULTILIB_WRAPPED_HEADERS=(
 	/usr/include/srtp/config.h
 )
-PATCHES=( "${FILESDIR}/${PN}-pcap-automagic-r0.patch" )
+PATCHES=( "${FILESDIR}/${PN}-pcap-automagic-r0.patch" "${FILESDIR}/${P}-openssl-1.1.patch" )
```

Comment:
Are you sure you want to add a patch to a stable ebuild in the same revision? Bear in mind that >=dev-libs/openssl-1.1 (for which this patch is made) is still not stable and even masked.

Reply:
That was a mistake. My apologies for the trouble!